pkg/render/fluentd.go

```diff
@@ -299,9 +304,11 @@ func (c *fluentdComponent) Objects() ([]client.Object, []client.Object) {
 		objs = append(objs, c.fluentdPodSecurityPolicy())
 	}
 
-	objs = append(objs, secret.ToRuntimeObjects(secret.CopyToNamespace(LogCollectorNamespace, c.cfg.ESSecrets...)...)...)
 	objs = append(objs, c.fluentdServiceAccount())
-	objs = append(objs, c.packetCaptureApiRole(), c.packetCaptureApiRoleBinding())
+	if c.cfg.EnterpriseVariant {
```

Comment:
Please change this to be:

```go
if len(c.cfg.ESSecrets) >0 {
  objs = append(objs, secret.ToRuntimeObjects(secret.CopyToNamespace(LogCollectorNamespace, c.cfg.ESSecrets...)...)...)
}
```

Otherwise the caller may pass ESSecrets and not understand why they aren't being respected.